BombsExcersice/BombsExcersice.cpp

```diff
@@ -1,10 +1,173 @@
-// BombsExcersice.cpp : This file contains the 'main' function. Program execution begins and ends there.
-//
-
 #include <iostream>
+#include <vector>
+#include <fstream>
+#include <string>
+
+enum Type {Empty, Bomb, Wall};
+
+class Cell
+{
+public:
+    Cell() : type{ Empty }, lifetime{ 0 }, maxLifetime{ 3 } {}
```

Comment:
Since `3` is a pretty specific number set by the task, I would place it outside of the class. You can do it via `setMaxLifeTime(int time)` or just make it a constructor parameter. With this change, it would be easier to change the code, if each bomb had its own specific lifetime.